### PR TITLE
Create integration.yml

### PR DESCRIPTION
**Pull request by nataliepina**
## Checklist for PR creator

- [ ] If any new components were added, there are exported from `packages/index.ts`
- [ ] If this PR is associated with a JIRA, it is mentioned in commit message footer ("Closes …")
- [ ] If this PR contains breaking changes, is stated in commit message body ("BREAKING CHANGE: …")
- [x] Info for applicable sections above is provided

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install node
+        uses: actions/setup-node@v2
+        with:
+          node:version: "16"
+      - name: Install dependencies
+        run: npm install
+      - name: Build
+        run: npm run dist
```
Comment (vacas5) [inline .github/workflows/integration.yml:16]: do you need to build storybook instead of the `dist` folder?

Comment (nataliepina) [reply .github/workflows/integration.yml:16]: It has thia before the integrations tests in the jenkinsfile 
```
      stage('Build') {
          steps {
            ansiColor('xterm') {
              sh '''npm run dist'''
            }
          }

          post {
            always {
              archiveArtifacts 'dist/**/*'
            }
          }
        }
        ```
I wrestled with this test for quite a while and it didn't work unless I had this build action with npm run dist. The `test-integration` file for the `npm run test:integration` takes care of running storybook and then a `npx cypress run`.  Open to suggestions if you have any ideas!

Comment (nataliepina) [reply .github/workflows/integration.yml:16]: I wrestled with this test for quite a while and it didn't work unless I had this build action with npm run dist. The `test-integration` file for the `npm run test:integration` takes care of running storybook and then a `npx cypress run`.  Open to suggestions if you have any ideas!

Comment (vacas5) [reply .github/workflows/integration.yml:16]: no it must rely on it under the hood. Sorry for the misdirection

**Review by vacas5 — APPROVED** [review]

**Pull request merged**